File: multi-gammon-core/src/main/java/org/akoshterek/backgammon/eval/Gammons.java
package org.akoshterek.backgammon.eval;

public interface Gammons {
    /* gammon possible by side on roll */
    int G_POSSIBLE = 0x1;

    /* backgammon possible by side on roll */
    int BG_POSSIBLE = 0x2;

    /* gammon possible by side not on roll */
    int OG_POSSIBLE = 0x4;

    /* backgammon possible by side not on roll */
    int OBG_POSSIBLE = 0x8;
}

File: multi-gammon-core/src/main/java/org/akoshterek/backgammon/eval/Gammons.scala
package org.akoshterek.backgammon.eval

object Gammons {
  /* gammon possible by side on roll */
  val G_POSSIBLE: Int = 0x1
  /* backgammon possible by side on roll */
  val BG_POSSIBLE: Int = 0x2
  /* gammon possible by side not on roll */
  val OG_POSSIBLE: Int = 0x4
  /* backgammon possible by side not on roll */
  val OBG_POSSIBLE: Int = 0x8
}